WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -282,7 +231,7 @@ public static void uploadMedia(Context context, @NonNull ArrayList<MediaModel> m
      * waiting for media to finish uploading counts as 'waiting to be uploaded' until the media uploads complete.
      */
     public static boolean isPostUploadingOrQueued(PostModel post) {
-        if (post == null) {
+        if (sInstance == null || post == null) {
```

Comment:
we can probably make this a oneliner like you did a few lines below, in `isPostQueued()`

Reply:
I wanted to, but I think in this case what's going on is complicated enough that it's worth spacing it out and having the explanatory comments. Will defer to your opinion though - do you think we should just collapse that into one big comment and one-line the check?

Reply:
oh you're right - the `||` operand here makes a difference. Let's leave it as is 👍